tdrs-backend/tdpservice/test/test_clamav.py

```diff
@@ -0,0 +1,74 @@
+"""Integration test(s) for clamav-rest operations."""
+from io import StringIO
+import pytest
+import requests
+
+from django.conf import settings
+from factory.faker import faker
+
+_faker = faker.Faker()
+
+
+@pytest.fixture
+def clamav_url():
+    """URL that can be used to reach ClamAV-REST."""
+    return settings.AV_SCAN_URL
+
+
+@pytest.fixture
+def fake_file_name():
+    """Generate a random, but valid file name ending in .txt."""
+    return _faker.file_name(extension='txt')
+
+
+@pytest.fixture
+def fake_file():
+    """Generate an in-memory file-like object with random contents."""
+    return StringIO(_faker.sentence())
+
+
+@pytest.fixture
+def infected_file():
+    """Generate an EICAR test file that will be treated as an infected file.
+
+    https://en.wikipedia.org/wiki/EICAR_test_file
+    """
+    return StringIO(
+        'X5O!P%@AP[4\\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*'
```

Comment:
```suggestion
        'X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*'
```
minor edit here results in the following expected result:
`{ Status: "FOUND", Description: "Win.Test.EICAR_HDB-1" }`

Reply:
and actually should have raised this as a question: i needed to make this change for testing locally, but is this sort of change needed otherwise?

Reply:
Ah, I added that extra backslash to prevent this IDE error:
![Screenshot from 2021-04-29 00-46-32](https://user-images.githubusercontent.com/22626085/116503558-6cd54e00-a884-11eb-94b3-4c3419d033f2.png)
Along with resolving this warning from the test output:
```
tdpservice/test/test_clamav.py:37
  /tdpapp/tdpservice/test/test_clamav.py:37: DeprecationWarning: invalid escape sequence \P
    'X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*'
```
However, looks like this may be another Windows vs. Linux difference - it doesn't seem like Windows ignores the extra backslash in the same way Linux does so the file isn't being matched correctly. Using a [raw string](https://www.journaldev.com/23598/python-raw-string) here resolves both the IDE and console warning on Linux. I pushed an update that implements that instead - can you pull down and re-test to confirm it also works as expected on Windows?

Reply:
confirming that this works on windows:
`{ Status: "FOUND", Description: "Eicar-Signature" }`
 (p.s. earlier test result was from my mac)